parmed/openmm/parameters.py

```diff
@@ -101,10 +107,20 @@ def from_parameterset(cls, params, copy=False):
         new_params.parametersets = params.parametersets
         new_params._combining_rule = params.combining_rule
         new_params.residues = params.residues
+        # Assign Lennard-Jones atom classes
+        ljtypes = dict()
+        counter = 0
+        for name, atom_type in iteritems(new_params.atom_types):
+            key = (atom_type.rmin, atom_type.epsilon, atom_type.rmin_14,
+                    atom_type.epsilon_14)
+            if key not in ljtypes:
+                ljtypes[key] = counter
```

Comment:
> That said, taking one of the atom type names should work nicely. I'll probably just keep the first type corresponding each L-J parameter type as the class name.

Maybe we should think a bit more deeply about the most obvious file we would want to mix-and-match: [`gaff.dat`](https://github.com/choderalab/ambermini/blob/master/share/amber/dat/leap/parm/gaff.dat), perhaps mixed with another one of the AMBER protein forcefield types.

`gaff.dat` defines [separate LJ parameters for each atom type](https://github.com/choderalab/ambermini/blob/master/share/amber/dat/leap/parm/gaff.dat#L5891-L5962), even if they are duplicates.  This means, as you observe, that using compressed numerical groups wouldn't let us mix-and-match due to collisions, but either the compressed types written as corresponding atom type names or writing them all by atom name would work.

Is there any benefit in adding a flag to select which scheme we want?  Or should we just figure out the optimal way and stick with that?